PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,26 @@
 <!--
 Thanks for your contribution!
```

Comment:
i love the changes, but your commit message cooouuld use some work. make it

Revise ``Pull Request`` template
Fixes typos, formatting and wording.